legend-sdlc-server-fs/src/main/java/org/finos/legend/sdlc/server/startup/BaseModule.java

```diff
@@ -0,0 +1,90 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.startup;
+
+import com.google.inject.Binder;
+import org.finos.legend.sdlc.server.api.backup.FileSystemBackupApi;
+import org.finos.legend.sdlc.server.api.build.FileSystemBuildApi;
+import org.finos.legend.sdlc.server.api.comparison.FileSystemComparisonApi;
+import org.finos.legend.sdlc.server.api.conflictresolution.FileSystemConflictResolutionApi;
+import org.finos.legend.sdlc.server.api.entity.FileSystemEntityApi;
+import org.finos.legend.sdlc.server.api.issue.FileSystemIssueApi;
+import org.finos.legend.sdlc.server.api.patch.FileSystemPatchApi;
+import org.finos.legend.sdlc.server.api.project.FileSystemProjectApi;
+import org.finos.legend.sdlc.server.api.project.FileSystemProjectConfigurationApi;
+import org.finos.legend.sdlc.server.api.review.FileSystemReviewApi;
+import org.finos.legend.sdlc.server.api.revision.FileSystemRevisionApi;
+import org.finos.legend.sdlc.server.api.user.FileSystemUserApi;
+import org.finos.legend.sdlc.server.api.version.FileSystemVersionApi;
+import org.finos.legend.sdlc.server.api.workflow.FileSystemWorkflowApi;
+import org.finos.legend.sdlc.server.api.workflow.FileSystemWorkflowJobApi;
+import org.finos.legend.sdlc.server.api.workspace.FileSystemWorkspaceApi;
+import org.finos.legend.sdlc.server.depot.FileSystemMetadataApi;
+import org.finos.legend.sdlc.server.depot.api.MetadataApi;
+import org.finos.legend.sdlc.server.domain.api.backup.BackupApi;
+import org.finos.legend.sdlc.server.domain.api.build.BuildApi;
+import org.finos.legend.sdlc.server.domain.api.comparison.ComparisonApi;
+import org.finos.legend.sdlc.server.domain.api.conflictResolution.ConflictResolutionApi;
+import org.finos.legend.sdlc.server.domain.api.entity.EntityApi;
+import org.finos.legend.sdlc.server.domain.api.issue.IssueApi;
+import org.finos.legend.sdlc.server.domain.api.patch.PatchApi;
+import org.finos.legend.sdlc.server.domain.api.project.ProjectApi;
+import org.finos.legend.sdlc.server.domain.api.project.ProjectConfigurationApi;
+import org.finos.legend.sdlc.server.domain.api.review.ReviewApi;
+import org.finos.legend.sdlc.server.domain.api.revision.RevisionApi;
+import org.finos.legend.sdlc.server.domain.api.user.UserApi;
+import org.finos.legend.sdlc.server.domain.api.version.VersionApi;
+import org.finos.legend.sdlc.server.domain.api.workflow.WorkflowApi;
+import org.finos.legend.sdlc.server.domain.api.workflow.WorkflowJobApi;
+import org.finos.legend.sdlc.server.domain.api.workspace.WorkspaceApi;
+import org.finos.legend.sdlc.server.resources.FileSystemAuthCheckResource;
+import org.finos.legend.sdlc.server.resources.FileSystemAuthResource;
+
+public class BaseModule extends AbstractBaseModule
```

Comment:
BaseModule is not the right name for this class. That name implies that it's intended to be extended, but I don't think that's the case.